packages/core/source/elements/RemoteElement.ts

```diff
@@ -268,7 +268,7 @@ export abstract class RemoteElement<
   Methods extends Record<string, (...args: any[]) => any> = {},
   Slots extends Record<string, any> = {},
   EventListeners extends Record<string, any> = {},
-> extends HTMLElement {
+> extends globalThis.HTMLElement {
```

Comment:
As long as we have a local scope for globalThis in realm, this should use the correct one.